python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1484,6 +1486,24 @@ keys:
                       - str
                     min: 2
                     max: 64
+          evpn_ethernet_segment:
```

Comment:
this should go above the `bgp_additional_paths:` key as it is marked for removal